src/html/components/title.html

```diff
@@ -2,11 +2,13 @@
.2 3.4 8 0 14.6-6.5 14.6-14.5l.1-48.8c0-8-6.5-14.6-14.5-14.6z" opacity=".53" fill="#b26860"/></g><g class="title-tablet" data-module="pills"><circle cx="327.3" cy="40.4" r="40.4" fill="#966992"/><path d="M355.7 11.5c13.8 17.6 10.6 43-6.9 56.8-16 12.6-38.9 11.2-53.4-3.2 13.8 17.6 39.2 20.7 56.8 6.9s20.7-39.2 6.9-56.8c-1.1-1.2-2.2-2.5-3.4-3.7z" opacity=".6" fill="#785989"/><path d="M342 3l-40 69.1c1.6 1.3.9 1.2 2.8 2.2s1.4.5 3.3 1.3L348 6.5c-1.6-1.3-.9-1.2-2.8-2.2S344 3.8 342 3z" opacity=".6" fill="#624a72"/></g><path d="M10.2 2.7h8.9v77h-8.9v-77zm76.2 60l27.7-60h11.3v77h-8.6V15.6l-26 57.1H82L55.9 15.6v64.1h-8.6v-77h11.4l27.7 60zm67.4-60H175c14 0 23 8.2 23 22.2s-9.5 22.6-23 22.6h-12.3v32.2h-8.9v-77zm20.4 36.9c9.2 0 14.6-5.7 14.6-14.6s-5.4-14.3-14.6-14.3h-11.4v28.9h11.4zm46.5-36.9h19.6c15.7 0 24 7.9 24 21.3 0 9.1-4.5 17.5-13.4 20.3L270 79.7h-9.9l-17.4-33.4h-13.1v33.4h-8.9v-77zm19.6 35.8c9 0 14.6-4.2 14.6-14.3 0-8.1-4.3-13.5-15.2-13.5h-10.1v27.8h10.7zm167.8 41.2l-29.3-77h9.8l24.2 66.4 24.3-66.4h9.8l-29.3 77h-9.5zm59.6-77h8.9v77h-8.9v-77zm37.1 0h8.9l42.5 62.9V2.7h8.8v77h-8.8L513.8 17v62.7h-8.9v-77h-.1zm151.5 9.8l-5.7 5.9c-3.5-3.4-10.3-8.8-22.4-8.8-17.7 0-29.8 13.6-29.8 31.7 0 18.6 12.5 31.6 29.7 31.6 12.9 0 20.1-5.6 22.5-8V48.7h-17.2v-7.8h25.7v27.8c-8.4 8.5-18.4 12.4-31.5 12.4-21.9 0-38.7-16.4-38.7-39.8 0-22.5 16.4-40 38.9-40 15.4 0 24.5 7.2 28.5 11.2zm-531 170.2l27.7-59.9h11.3v77h-8.6v-64.1l-26.1 57.1h-8.8l-26.1-57.1v64.1h-8.6v-77h11.4l27.8 59.9zm67.4-60h36.5v8.1h-27.6v25.6h24.1v8h-24.1v27.2h28.8v8h-37.7v-76.9zm63.2 0h15.8c26.3 0 41.2 14.4 41.2 38.5 0 25.5-15.5 38.5-41.2 38.5h-15.8v-77zm17.6 69.1c18.5 0 30.4-10.4 30.4-30.6 0-19.2-11.9-30.5-30.4-30.5h-8.7v61.1h8.7zm171.2.6c-7 5.8-15.8 8.9-24.9 8.8-22.5 0-39-17.2-39-39.9s17-39.9 38.8-39.9c8.6-.1 17 2.7 23.9 8l-2.9 7.9c-5.9-4.8-13.2-7.5-20.8-7.6-17 0-29.8 13.6-29.8 31.6s12.2 31.6 30 31.6c8.1 0 15.8-3 21.9-8.2l2.8 7.7zm34.6-17.6l-9.2 25h-9l28.8-77h9l28.8 77h-9l-9.2-25h-30.2zm15-42.3L482 167.1h24.8l-12.5-34.6zm54.5-9.8h8.9v69h28.8v8h-37.7v-77zM18.3 294.8l-9.3 25H0l28.8-77h9l28.8 77h-9l-9.2-25H18.3zm15-42.3L21 287.1h24.8l-12.5-34.6zm54.5-9.8h15.8c26.3 0 41.2 14.4 41.2 38.5 0 25.5-15.5 38.5-41.2 38.5H87.8v-77zm17.6 69.1c18.5 0 30.4-10.5 30.4-30.6 0-19.2-11.9-30.5-30.4-30.5h-8.7v61h8.7v.1zm63.5-69.1h8.9v33.8h40.9v-33.8h8.9v77h-8.9v-35.1h-40.9v35.1h-8.9v-77zm87.1 0h36.5v8.1h-27.6v25.6H289v8h-24.1v27.2h28.8v8H256v-76.9zm63.1 0h19.6c15.7 0 24 7.9 24 21.3 0 9.1-4.5 17.5-13.4 20.4l19.1 35.3h-9.9l-17.4-33.4H328v33.4h-8.9v-77zm19.6 35.8c9 0 14.6-4.2 14.6-14.3 0-8.1-4.3-13.5-15.2-13.5H328v27.8h10.7zm51.6-35.8h36.5v8.1h-27.6v25.6h24.1v8h-24.1v27.2H428v8h-37.7v-76.9zm63 0h8.9l42.5 62.9v-62.9h8.8v77h-8.8L462.2 257v62.7h-8.9v-77zm148.1 69.7c-7 5.8-15.8 8.9-24.9 8.8-22.5 0-39-17.2-39-39.9s17-39.9 38.8-39.9c8.6-.1 17 2.7 23.9 8l-2.9 7.9c-5.9-4.8-13.2-7.5-20.8-7.6-17 0-29.8 13.6-29.8 31.6s12.2 31.6 30 31.6c8-.1 15.8-3 21.9-8.2l2.8 7.7zm23.6-69.7h36.5v8.1h-27.6v25.6H658v8h-24.1v27.2h28.8v8H625v-76.9z" fill="#184e67"/></svg>
```

Comment:
You could try using nunjucks include tag to add the graphic to the template, e.g.:
```
{% include "../../images/title.svg" %}
```